.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,6 +11,4 @@ Provide a few more details like what files were modified, added or removed.
 * What was modified:
 * What was added:
 * What was removed:
-* Next actions:
 
```

Comment:
Maybe one more section for how to best review?

```suggestion
## How to review
```

Reply:
👍